acceptance/src/data.rs

```diff
@@ -94,6 +84,33 @@ fn assert_schema_fields_match(schema: &Schema, golden: &Schema) {
     }
 }
 
+// some things are equivalent, but don't show up as equivalent for `==`, so we normalize here
+fn normalize_col(col: Arc<dyn Array>) -> Arc<dyn Array> {
+    if let DataType::Timestamp(unit, Some(zone)) = col.data_type() {
+        if **zone == *"+00:00" {
+            arrow_cast::cast::cast(&col, &DataType::Timestamp(unit.clone(), Some("UTC".into())))
```

Comment:
```suggestion
            arrow_cast::cast::cast(&col, &DataType::Timestamp(*unit, Some("UTC".into())))
```

I think clippy is yelling since it implements `Copy`?

Reply:
yeah, that changed that from `52.0` (which I had locally) to `52.1`, so I had to `cargo update` to get the lint :)